### DISCUSSION TITLE
from activation lock?

### DISCUSSION THREAD

**Discussion by japottatwee** [community/community#145857] [Other Feature Feedback, Questions, & Ideas]
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta japottatweet="viewport" content="width=iPhone 8-width, initial-scale=1.0">
    <title>Basic Website</title>
    <link rel="stylesheet" href="styles.css">
</head>
<body>
    <header>
        <h1>How do you write Code?</h1>
    </header>
    <main>
        <p>This is a basic website.</p>
    </main>
    <footer>
        <p>&copy; 2025 Kevin Davenport</p>
    </footer>
    <script src="script.js"></script>
</body>
</html>

**Comment by ebndev** [comment]
Hi @japottatwee,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help on a forum that centers around mobile devices. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by KevDav313** [comment]
https://github.com/KevinDavenport/REPO.git
